### ISSUE TITLE
AlamofireObjectMapper 6 + Alamofire 4 (latest final versions) not compatible

### ISSUE THREAD

**Issue by niccey** [tristanhimmelman/AlamofireObjectMapper#264]
It seems to me that the latest final release AlamofireObjectMapper 6 is not (downward) compatible with the latest final version of Alamofire (4; 5 is still beta).

So this combination using Carthage has stopped working since:
```
github "Alamofire/Alamofire"
github "tristanhimmelman/AlamofireObjectMapper"
```

This, however, works:
```
github "Alamofire/Alamofire"
github "tristanhimmelman/AlamofireObjectMapper" ~> 5.2
```

Is this known and accepted? Shouldn't a new final version of AlamofireObjectMapper be compatible with the latest final version of Alamofire (it will soon presumably, but not yet)?

By the way, thank you for providing this and other great libraries!

**Comment by rzulkoski** [comment]
@niccey, I've created a fork that is compatible with Xcode 10.2. It upgrades the Swift version from 3.0 to 4.0.

Try putting this in your Cartfile:
```
github "rzulkoski/AlamofireObjectMapper" "migrate-to-swift-4"
```

@tristanhimmelman, is it possible to merge my changes in a hotfix release? Say `5.3.0`? Here's my [branch](https://github.com/rzulkoski/AlamofireObjectMapper/tree/migrate-to-swift-4).

**Comment by kublaios** [comment]
> @niccey, I've created a fork that is compatible with Xcode 10.2. It upgrades the Swift version from 3.0 to 4.0.
> 
> Try putting this in your Cartfile:
> 
> ```
> github "rzulkoski/AlamofireObjectMapper" "migrate-to-swift-4"
> ```
> 
> @tristanhimmelman, is it possible to merge my changes in a hotfix release? Say `5.3.0`? Here's my [branch](https://github.com/rzulkoski/AlamofireObjectMapper/tree/migrate-to-swift-4).

Thank you, that was the only way I could get around `dyld: Symbol not found` error. I hope the repo gets maintained soon.

**Comment by niccey** [comment]
@kublaios That's right, with Xcode 10.2 that is. I ran into the same problem now. My above Cartfile definition worked until Xcode 10.1.

Thanks, @rzulkoski. Your solution is not directly related to my OP but it solves this new Xcode 10.2 problem. With Xcode 10.2 and without your branch, I couldn't even get it to work with AlamofireObjectMapper 6 + Alamofire 5 Beta, I think I got the error mentioned by @kublaios. I guess this issue affects many users of this library.

**Comment by rzulkoski** [comment]
Yeah, #266 references the same issue and has a similar fix. Whenever Alamofire 5 is officially released we should all be able to go back to master, but I'll be using this fork in the meantime. Glad you guys were able to get back up and running. /cheers

**Comment by RomanPodymov** [comment]
Hello @niccey 
I think you should try AlamofireObjectMapper v.5.2.1. It uses Alamofire 4.8.2.
